### PR TITLE
Update site content and images

### PR DESCRIPTION
**Pull request by KaitCrawford**
we've received new content and images for the site

The proper styling will be done as part of another ticket

### CODE REVIEW

File: pages/about.js [hunk 1]
```diff
@@ -8,18 +8,18 @@ export default function About() {
...
           <p className={utilStyles.description}>
-            With countries experiencing a massive need for direct communication about COVID-19, {' '}
-            <a href="https://www.praekelt.org/">Praekelt.org</a> (using <a href="https://www.turn.io/">Turn.io</a>)
-            has developed a pro bono solution to empower citizens with the right
-            information and support the national
-            response of the health systems that serve them.
+            With countries experiencing a massive need for direct communication
+            about COVID-19, Praekelt.org (using Turn.io) has developed a pro bono
+            solution to empower citizens with the right information and support
+            the national response of the health systems that serve them.
```
Comment (codiebeulaine) [inline pages/about.js:18]: how come we are removing the links?

**Review by codiebeulaine — APPROVED** [review]
Small question but looks good 👍

File: pages/about.js (hunk 1 above)
Comment (KaitCrawford) [reply pages/about.js:18]: Thanks for pointing that out. They aren't in the copy but I'll check if they are supposed to be there

Comment (KaitCrawford) [reply pages/about.js:18]: Confirmed that these links shouldn't be here.

**Pull request merged**